addons/website/views/snippets/snippets.xml

```diff
@@ -1071,7 +1071,7 @@
     <div data-js="ThemeColors" data-selector="theme-colors" data-no-check="true">
         <we-alert class="o_old_color_system_warning d-none">
             It appears your website is still using the old color system of
-            Odoo 13.0 in some places. We made sure it is still working but
+            Odoo 14.0 in some places. We made sure it is still working but
```

Comment:
Given the surrounding text, it appears that it should stay "13.0" in this case.

Reply:
@AntoineVDV  Updated